presto-main/src/main/java/com/facebook/presto/operator/TaskContext.java

```diff
@@ -573,11 +572,11 @@ public TaskStats getTaskStats()
         boolean fullyBlocked = hasRunningPipelines && runningPipelinesFullyBlocked;
 
         return new TaskStats(
-                taskStateMachine.getCreatedTime(),
-                executionStartTime.get(),
-                lastExecutionStartTime.get(),
-                lastExecutionEndTime == 0 ? null : new DateTime(lastExecutionEndTime),
-                executionEndTime.get(),
+                taskStateMachine.getCreatedTimeInMillis(),
+                Optional.ofNullable(executionStartTime.get()).orElse(0L),
```

Comment:
we are already defining it above as `AtomicReference<Long> executionStartTime = new AtomicReference<>();`....will it be null ever ? do we need Optional.ofNullable ?